Example/NativeBase-KitchenSink/src/screens/toast/toast-duration.js

```diff
@@ -42,7 +43,10 @@ class ToastDuration extends Component {
               Toast.show({
                 text: "Wrong password!",
                 buttonText: "Okay",
-                duration: 3000
+                duration: 3000,
+                onClose: () => {
```

Comment:
Is there a way to capture automatically? So customer does not need to add one line in onClose?